platform-plugins.md

```diff
@@ -210,6 +210,59 @@ The iOS platform code of your plugin is located in `Pods/Development Pods/hello/
 
 You can run the example app by pressing the &#9654; button.
 
+#### iOS platform code (.swift)
+
+Flutter tooling currently expects iOS plugins to expose an Objective-C header. But you can
+implement your plugin in Swift instead and just have the Objective-C part forward plugin
+registration to that implementation. This section describes how to change the Objective-C
+plugin generated by `flutter create --plugin hello` to a Swift-based implementation.
+
+Before editing the iOS platform code in Xcode, first make sure that
+the code has been built at least once by Flutter by executing
+`cd hello/example; flutter build ios`.
+
+Next,
+
+1. Launch Xcode
+1. Select 'File > Open', and select the `hello/example/ios/Runner.xcworkspace` file.
+   The iOS platform code of your plugin is located in `Pods/Development Pods/hello/Classes/` in the Project Navigator.
+1. Select the `Classes` folder, then select `File > New > File...` and add a new Cocoa Touch class.
+   Name the class `RealHelloPlugin`, make it a sub-class of `NSObject`, and select `Swift` as language.
+   Press `Next` and accept the location for the new class.
+1. Replace the code for the new class with
+    ```swift
+    import Flutter
+    import UIKit
+    
+    public class RealHelloPlugin: NSObject, FlutterPlugin {
+        public static func register(with registrar: FlutterPluginRegistrar) {
+            let channel = FlutterMethodChannel(name: "hello", binaryMessenger: registrar.messenger());
+            let instance = RealHelloPlugin();
+            registrar.addMethodCallDelegate(instance, channel: channel);
+        }
+        
+        public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+            result("iOS " + UIDevice.current.systemVersion);
+        }
+    }
+    ```
+1. Replace the code of `HelloPlugin.m` with a forwarding call:
+    ```objective-c
+    #import "HelloPlugin.h"
+    #import <hello/hello-Swift.h>
+    
+    @implementation HelloPlugin
+    + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
+        [RealHelloPlugin registerWithRegistrar:registrar];
+    }
+    @end
+    ```
+1. Build the project again using Flutter by executing `flutter build ios` in
```

Comment:
Yes, but you cannot launch from within Xcode without having done `pod install` again. Which happens when you do `flutter build ios`. I'll look at a way to rephrase to something simpler